packages/modules/store/src/models/currency.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  DALUtils,
+  Searchable,
+  createPsqlIndexStatementHelper,
+  generateEntityId,
+} from "@medusajs/utils"
+
+import {
+  BeforeCreate,
+  Entity,
+  OnInit,
+  PrimaryKey,
+  Property,
+  Filter,
+  ManyToOne,
+} from "@mikro-orm/core"
+import Store from "./store"
+
+const StoreCurrencyDeletedAtIndex = createPsqlIndexStatementHelper({
+  tableName: "store_currency",
+  columns: "deleted_at",
+  where: "deleted_at IS NOT NULL",
+})
+
+@Entity()
+@Filter(DALUtils.mikroOrmSoftDeletableFilterOptions)
+export default class StoreCurrency {
+  @PrimaryKey({ columnType: "text" })
+  id: string
+
+  @Searchable()
+  @Property({ columnType: "text" })
+  currency_code: string
+
+  @Property({ columnType: "boolean", default: true })
```

Comment:
@olivermrbl I am fine either way. I just checked, 102 countries use tax-inclusive pricing, notably most of europe, china, japan, and india. I think the biggest market where it's tax-exclusive is the US, so it depends which markets we want to optimize for. Let me know what you think.

Reply:
Source: https://help.shopify.com/en/manual/international/pricing/dynamic-tax-inclusive-pricing

Reply:
I don't have any strong opinion on this, and I think your argument on target market is reasonable. My thought was rooted in the fact that most other commerce platforms come with this feature disabled by default, so for users migrating to Medusa, this could be unexpected behavior. That said, this is a very minor thing and can be solved with proper education. 

Will let @srindom decide on this one.

Reply:
I removed the is_tax_inclusive flag as it will be moved to the pricing module, resolving the comment for now (we can re-discuss once we have it implemented again)